### ISSUE TITLE
Battery Health - LFP - Status: 500. Message: convert frame from rows error: pq: division by zero

### ISSUE THREAD

**Issue by henzay** [teslamate-org/teslamate#3675]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

I am receiving an error when I use the battery health LFP screen in the graph "Battery Capacity by Mileage"

### Expected Behavior

Normally a red line will be displayed to show the batttery capacity degradation. 
here it is not displayed. 

### Steps To Reproduce

Go to the Battery Health - LFP screen 

### Relevant log output

```shell
teslamate-1  | 2024-02-14 08:45:03.356 [info] GET /
teslamate-1  | 2024-02-14 08:45:03.361 [info] Sent 200 in 4ms
teslamate-1  | 2024-02-14 10:28:16.281 [info] Refreshing access token ...
teslamate-1  | 2024-02-14 10:28:16.761 [info] POST https://auth.tesla.com/oauth2/v3/token -> 200 (479.380 ms)
teslamate-1  | 2024-02-14 10:28:16.764 [info] Scheduling token refresh in 6 h
teslamate-1  | 2024-02-14 13:47:21.412 [info] GET /
teslamate-1  | 2024-02-14 13:47:21.419 [info] Sent 200 in 6ms
teslamate-1  | 2024-02-14 14:06:42.560 [info] GET /
teslamate-1  | 2024-02-14 14:06:42.566 [info] Sent 200 in 6ms
```

### Screenshots

<img width="1304" alt="Screenshot 2024-02-14 at 15 08 37" src="https://github.com/teslamate-org/teslamate/assets/9012944/9d9652ca-d368-4097-bd5e-eebb470d8941">

### Type of installation

Docker

### Version

v1.28.3

**Comment by JakobLichterfeld** [comment]
Sorry, my bad, #3593 is already included in 1.28.3

@jheredianet can you please take a look into this?

**Comment by jheredianet** [comment]
> Sorry, my bad, #3593 is already included in 1.28.3
> 
> @jheredianet can you please take a look into this?

Hi, sorry, I can only speak on the original "Battery Health" dashboard as the creator of it. 
As I've explained in [other post](https://github.com/teslamate-org/teslamate/discussions/3513#discussioncomment-7694883), IMHO, there is no need to have another battery health dashboard for LFP batteries. 

Somebody cloned the dashboard into LPF one, with "some adjustments" and his PR has been accepted as valid, but then it has not carried out the respective maintenance, since the original dashboard has been modified and adjusted to avoid calculation errors and null values and take care of different units of measurement. 

So, @henzay I guess, the error "divided by 0" on the original "Battery Health" dashboard is not happening. If so, that's the dashboard you would need to use. 

I've tried to remove the [LFP dashboards](https://github.com/teslamate-org/teslamate/pull/3531) in a PR, but it was denied. The best option is to [implement this feature](https://github.com/teslamate-org/teslamate/issues/3438).

**Comment by JakobLichterfeld** [comment]
> Hi, sorry, I can only speak on the original "Battery Health" dashboard as the creator of it. As I've explained in [other post](https://github.com/teslamate-org/teslamate/discussions/3513#discussioncomment-7694883), IMHO, there is no need to have another battery health dashboard for LFP batteries.

Thank you for jogging my memory.
So we need to maintain both dashboards until #3438 is not solved.
